### ISSUE TITLE
[feature] Ignored credentials temporary cache

### ISSUE THREAD

**Issue by QGIS-feature-tracker** [qgis/QGIS-Documentation#4352]
Original commit: https://github.com/qgis/QGIS/commit/7b38ca8b794fc4763ce21261cf6d92beafc0339e by web-flow

[feature] Ignored credentials temporary cache

**Comment by felixfeckler** [comment]
@DelazJ you got an opinion on where this should be documented?
Thought about https://docs.qgis.org/3.22/en/docs/user_manual/introduction/project_files.html and adding a subitem like: "Handling broken connections"

**Comment by DelazJ** [comment]
I don't think it deserves a new sub-section. This occurs in the same dialog as
https://docs.qgis.org/3.22/en/docs/user_manual/introduction/project_files.html#handling-broken-file-paths. Just one more option for DB so let's provide the info in it. You can tweak the title if you want.
BTW, if you could add a screenshot with unavailable data... Thanks.
